test/LaunchDarkly.EventSource.Tests/EventSourceTests.cs

```diff
@@ -205,11 +113,13 @@ public async Task Default_HTTP_method_is_get()
         public async Task HTTP_method_can_be_specified()
         {
             var handler = new StubMessageHandler();
-            handler.QueueStringResponse(":");
+            handler.QueueResponse(StubResponse.Ok());

```

Comment:
Some tests like this were sending a dummy comment like ":" just in order to provide a trigger for closing the EventSource immediately. Instead I'm using the new `RequestReceived` event which fires as soon as the mock serve has received a request.